Classes/Template.php

```diff
@@ -159,8 +159,11 @@ public function createOrFetchAndApply(NodeInterface $parentNode, array $context)
                 $node = $flowQuery->find($name)->get(0);
             }
             if (!$node instanceof NodeInterface) {
-                $node = $this->nodeOperations->create($parentNode, ['nodeType' => $this->type, 'nodeName' => $name],
-                    'into');
+                $type = $this->type;
+                if (preg_match(\Neos\Eel\Package::EelExpressionRecognizer, $type)) {
+                    $type = $this->eelEvaluationService->evaluateEelExpression($this->type, $context);
```

Comment:
```suggestion
                    $type = $this->eelEvaluationService->evaluateEelExpression($type, $context);
```
Again just a small suggestion - sorry for the nitpicking. ;)

Reply:
Haha whatever you deem necessary ;)

(i wouldnt be so critical about the code since well need to rewrite many parts either way i think ^^)